common/types/list.go

```diff
@@ -190,7 +190,12 @@ func (l *baseList) ConvertToNative(typeDesc reflect.Type) (any, error) {
 	// Allow the element ConvertToNative() function to determine whether conversion is possible.
 	otherElemType := typeDesc.Elem()
 	elemCount := l.size
-	nativeList := reflect.MakeSlice(typeDesc, elemCount, elemCount)
+	var nativeList reflect.Value
+	if typeDesc.Kind() == reflect.Array {
```

Comment:
I noticed along the way that arrays as opposed to lists are not handled correctly. If desired, I can move this into its own PR

Reply:
Thank you, that would be great if you could pull this into its own PR.

Reply:
https://github.com/google/cel-go/pull/894